validation/src/main/scala/hmda/validation/rules/lar/validity/V555.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object V555 extends EditCheck[LoanApplicationRegister] {
+
+  val loanPurposeTypes = List(1, 3)
+  val lienStatusTypes = List(1, 2, 4)
```

Comment:
I think I've seen this commented elsewhere as well, but the notation "Types" is potentially confusing, since we always talk about types in `Scala` to refer to the type system. I would read this variable and expect a type alias or something similar. Maybe change to `loanPurposeValues` or similar (look around in the codebase, not sure if we are already using some convention)